docker/templates/python/{{cookiecutter.directory_name}}/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Extend the base Python image
+# See https://hub.docker.com/_/python for version options
+# N.b., there are many options for Python images. We used the plain
+# version number in the pilot. YMMV. See this post for a discussion of
+# some options and their pros and cons:
+# https://pythonspeed.com/articles/base-image-python-docker-images/
+FROM python:3.7
+
+# Give ourselves some credit
+LABEL maintainer "DataMade <info@datamade.us>"
+
+# Install any additional OS-level packages you need via apt-get. RUN statements
+# add additional layers to your image, increasing its final size. Keep your
+# image small by combining related commands into one RUN statement, e.g.,
+#
+# RUN apt-get update && \
+#     apt-get install -y python-pip
+#
+# Read more on Dockerfile best practices at the source:
+# https://docs.docker.com/develop/develop-images/dockerfile_best-practices
+
+# Inside the container, create an app directory and switch into it
+RUN mkdir /app
+WORKDIR /app
+
+# Copy the requirements file into the app directory, and install them. Copy
+# only the requirements file, so Docker can cache this build step. Otherwise,
+# the requirements must be reinstalled every time you build the image after
+# the app code changes. See this post for further discussion of strategies
+# for building lean and efficient containers:
+# https://blog.realkinetic.com/building-minimal-docker-containers-for-python-applications-37d0272c52f3
+COPY ./requirements.txt /app/requirements.txt
```

Comment:
Should we store a `requirements.txt` file with the latest version of Django in this template? That way developers could immediately get started creating a Django project with `docker-compose run --rm --entrypoint django-admin app startproject`.

Reply:
Thought about this some more and realized that not all containerized Python projects will be Django projects, so maybe it would be better to just have an empty `requirements.txt` file (so that it can get mounted in the container correctly).

Reply:
upon further reflection, i think that these templates should be pretty tightly scoped to generating artifacts related to docker. if it's ok with you, i'll leave the requirements file out.

Reply:
hm, actually, i forgot docker expected a requirements file to exist. lemme walk that back...